src/tools/sample/sample.cpp

```diff
@@ -214,17 +329,32 @@ typedef struct QUIC_CREDENTIAL_CONFIG_HELPER {
     };
 } QUIC_CREDENTIAL_CONFIG_HELPER;
 
+//
+// Helper function to load a server configuration. Uses the command line
+// arguments to load the credential part of the configuration.
+//
 bool
 ServerLoadConfiguration(
     _In_ int argc,
     _In_reads_(argc) _Null_terminated_ char* argv[]
     )
 {
     QUIC_SETTINGS Settings{0};
+    //
+    // Configures the server's idle timeout.
+    //
     Settings.IdleTimeoutMs = IdleTimeoutMs;
     Settings.IsSet.IdleTimeoutMs = TRUE;
+    //
+    // Configures the server's resumption level to allow for resumption and
+    // 0-RTT.
+    //
     Settings.ServerResumptionLevel = QUIC_SERVER_RESUME_AND_ZERORTT;
     Settings.IsSet.ServerResumptionLevel = TRUE;
+    //
+    // Configures the server's settings to allow for the peer to open a single
+    // bidirectional stream.
```

Comment:
Mention here that setting some number of allowed streams is necessary otherwise the connection will immediately close after handshake.

Reply:
Well, it won't immediately close. It would eventually (1 second) just idle timeout. I'm not sure what else I really should say here. I could say, in an additional sentence, that "By default connections are not configured to allow any streams."

Reply:
Yeah that sounds good to me.